### DISCUSSION TITLE
Can't create a new Fork - is still open to contribution?

### DISCUSSION THREAD

**Discussion by IlSocio** [BrighterCommand/Brighter#2075] [Q&A] [answered]
I tried to fork the project and I got the error message below :(  Does exists a 230 maximum forks limit?

![image](https://user-images.githubusercontent.com/6009176/163190814-6bc7e8a7-b060-442d-bde6-c107e7966559.png)

**Comment by DevJonny** [comment, marked as answer]
I've just managed to create a fork on a separate account of mine (so now there are 231 forks) so this doesn't look to be an issue with the Brighter repository itself.

**Comment by IlSocio** [comment]
doh... 
I didn't notice that I already have forked the repo, so probably that's the reason because I got the error message :)
